src/coreComponents/unitTests/fieldSpecificationTests/CMakeLists.txt

```diff
@@ -13,6 +13,22 @@ else()
   list( APPEND dependencyList ${geosx_core_libs} )
 endif()
 
+if (TARGET pugixml::pugixml)
+  list( APPEND dependencyList pugixml::pugixml )
+endif()
+
+if (TARGET pugixml)
+  list( APPEND dependencyList pugixml )
+endif()
+
+if (TARGET fmt::fmt)
+  list( APPEND dependencyList fmt::fmt )
+endif()
+
+if (TARGET fmt)
+  list( APPEND dependencyList fmt )
+endif()
```

Comment:
@bmhan12 Why are there scoped and unscoped dependencies?

Reply:
> Why are there scoped and unscoped dependencies?

I found that the spack and the tplMirror built versions have different scopes for these two packages. This was added to support both build types.